SoftLayer/CLI/block/snapshot/enable.py

```diff
@@ -16,13 +16,16 @@
               help='Number of snapshots to retain.',
               required=True)
 @click.option('--minute',
-              help='Minute of the day when snapshots should be taken.',
+              help='Minute of the hour when snapshots should be taken, '
+              'integer between 0 to 59.',
```

Comment:
```suggestion
              help='Minute of the hour when snapshots should be taken, integer between 0 to 59.',
```
for this project the line length is 120 characters, so I prefer to keep things on one line if possible.